### DISCUSSION TITLE
Any suggestions for rendering large documents?

### DISCUSSION THREAD

**Discussion by thalissonbarbosas** [hrynko/vue-pdf-embed#48] [Q&A]
I'm getting memory issues when rendering a large PDF like 150+ pages.

Does anyone know how to do something like a lazily rendering?

Thanks

**Comment by hrynko** [comment]
Hi @thshiro,

There is a known issue with large documents in Safari that I'm currently working on. I expect to release a fix for single page view in `1.1.3`. This won't completely solve the problem, but at least it will allow the document to be paged out avoiding memory issues. I'm also considering implementing lazy loading (or rather lazy rendering), but this will most likely appear in the next release after `1.1.3`.
